seacrowd/sea_datasets/creole_rc/creole_rc.py

```diff
@@ -0,0 +1,197 @@
+
+    def _generate_examples(self, csv_filepath: Path, json_filepath: Path, split: str) -> Tuple[int, Dict]:
+        """Yields examples as (key, example) tuples."""
+        # read csv file
+        with open(csv_filepath, "r", encoding="utf-8") as csv_file:
+            csv_reader = csv.reader(csv_file)
+            csv_data = [row for row in csv_reader]
+        csv_data = csv_data[1:]  # remove header
+
+        # read json file
+        with open(json_filepath, "r", encoding="utf-8") as json_file:
+            json_data = json.load(json_file)
+
+        # properties descriptions from https://github.com/hclent/CreoleVal/tree/main/nlu/relation_classification
+        # for properties present in Chavacano subset
+        properties_desc = {"P17": "country", "P30": "continent", "P106": "occupation", "P131": "located in the administrative territorial entity", "P361": "part of", "P1376": " capital of country"}
+
+        num_sample = len(csv_data)
+
+        for i in range(num_sample):
+            if self.config.schema == "source":
+                example = {
+                    "id": str(i),
+                    "sentence": csv_data[i][0],
+                    "ent1": csv_data[i][1],
+                    "ent2": csv_data[i][2],
+                    "ent1_qcode": csv_data[i][3],
+                    "ent2_qcode": csv_data[i][4],
+                    "property": csv_data[i][5],
+                    "property_desc": properties_desc[csv_data[i][5]],
+                    "tokens": json_data[i]["tokens"],
+                    "edgeset_left": json_data[i]["edgeSet"]["left"],
+                    "edgeset_right": json_data[i]["edgeSet"]["right"],
+                    "edgeset_triple": json_data[i]["edgeSet"]["triple"],
+                }
+            elif self.config.schema == f"seacrowd_{self.SEACROWD_SCHEMA_NAME}":
+                offset_entity1 = csv_data[i][0].find(csv_data[i][1])
+                offset_entity2 = csv_data[i][0].find(csv_data[i][2])
+
+                example = {
+                    "id": str(i),
+                    "document_id": str(i),
+                    "passages": [{"id": f"passage-{i}", "type": "text", "text": [csv_data[i][0]], "offsets": [[0, len(csv_data[i][0])]]}],
+                    "entities": [
+                        {"id": f"{i}-entity-{csv_data[i][3]}", "type": "text", "text": [csv_data[i][1]], "normalized": [], "offsets": [[offset_entity1, offset_entity1 + len(csv_data[i][1])]]},
+                        {"id": f"{i}-entity-{csv_data[i][4]}", "type": "text", "text": [csv_data[i][2]], "normalized": [], "offsets": [[offset_entity2, offset_entity2 + len(csv_data[i][2])]]},
+                    ],
+                    "events": [],
+                    "coreferences": [],
+                    "relations": [{"id": f"{i}-relation-{csv_data[i][5]}", "type": properties_desc[csv_data[i][5]], "arg1_id": f"{i}-entity-{csv_data[i][3]}", "arg2_id": f"{i}-entity-{csv_data[i][4]}"}],
+                }
```

Comment:
missing `normalized` on this schema that leads to error on my end. Not sure why @faridlazuarda's test works on his end, prob due to earlier version of `datasets` module, tho. mine tested on `datasets==2.16.1`